src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -118,7 +124,7 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       return;
     }
 
-    const stateObj = this.hass.states[this._config.entity] as MediaEntity;
+    const stateObj = this._stateObj;
```

Comment:
Where is this defined? I feel like i'm missing it somewhere

Reply:
line 543.